packages/server/graphql/types/Story.ts

```diff
@@ -0,0 +1,18 @@
+import {GraphQLID, GraphQLInterfaceType, GraphQLNonNull} from 'graphql'
+
+export const storyFields = () => ({
+  id: {
+    type: GraphQLNonNull(GraphQLID),
+    description: 'serviceTaskId'
+  }
+})
+
+const Story = new GraphQLInterfaceType({
+  name: 'Story',
```

Comment:
@nickoferrall you probably beat me to it, but I needed a `Story` type for this, may be a merge conflict in our future